src/expr/src/relation/func.rs

```diff
@@ -191,7 +190,11 @@ where
         })
 }
 
-fn string_agg<'a, I>(datums: I, temp_storage: &'a RowArena, order_by: &[ColumnOrder]) -> Datum<'a>
+fn string_agg<'a, I>(
+    datums: I,
+    temp_storage: &'a RowArena,
+    order_by: &'a [ColumnOrder],
```

Comment:
This is not correct, the lifetime of `order_by` is independent of the datums.

Reply:
Thanks, fixed!